.pipelines/profiles.yml

```diff
@@ -24,4 +24,13 @@ default:
       schema: "{{ var('DBT_SCHEMA') }}"
       threads: 10
 
+    databricks-ci:
+      type: databricks
+      catalog: "{{ var('DBT_DATABRICKS_CATALOG') }}"
+      schema: "{{ var('DBT_SCHEMA') }}"
+      host: "{{ var('DBT_DATABRICKS_HOST') }}"
+      http_path: "{{ var('DBT_DATABRICKS_HTTP_PATH') }}"
+      token: "{{ var('DBT_DATABRICKS_TOKEN') }}"
+      threads: 10
+
   target: snowflake-ci
```

Comment:
I assume, like our normal transformations, from this point the target should be `databricks-ci`

Reply:
Maybe not necessary for how this project and the corresponding pipeline works, but still consistent.